src/main/java/org/mongojack/internal/util/SerializationUtils.java

```diff
@@ -196,13 +195,17 @@ private static Object serializeQueryCondition(
         } else {
             CompoundQueryCondition compound = (CompoundQueryCondition) condition;
             if (!key.startsWith("$")) {
```

Comment:
can you change this to use `isOperator` as well

Reply:
Done